src/explore-education-statistics-common/src/modules/table-tool/components/__tests__/PublicationForm.test.tsx

```diff
@@ -389,28 +395,26 @@ describe('PublicationForm', () => {
 
     await userEvent.type(screen.getByLabelText('Search publications'), 'Nope');
 
-    jest.runOnlyPendingTimers();
+    expect(
+      await screen.findByText('No publications found'),
+    ).toBeInTheDocument();
 
     const publicationRadios = within(
       screen.getByRole('group', {
         name: /Select a publication/,
       }),
     ).queryAllByRole('radio');
     expect(publicationRadios).toHaveLength(0);
-
-    expect(screen.getByText('No publications found')).toBeInTheDocument();
   });
 
   test('does not throw error if regex sensitive search term is used', async () => {
     render(
       <PublicationForm {...wizardProps} themes={testThemes} onSubmit={noop} />,
     );
 
-    await userEvent.type(screen.getByLabelText('Search publications'), '[');
+    await userEvent.type(screen.getByLabelText('Search publications'), '[[');
 
-    expect(() => {
-      jest.runOnlyPendingTimers();
-    }).not.toThrow();
+    expect(() => {}).not.toThrow();
```

Comment:
Does this line serve a purpose?

Reply:
Good spot :) I think I updated this test before I finally found out how to get Jest fake timers to work with userEvent. I've updated it now so it's like the previous version of the test.